frontend/model/contracts/group.js

```diff
@@ -298,7 +298,7 @@ DefineContract({
           }
 
           if (deepEqualJSONType(omit(prop.data.proposalData, 'reason'), dataToCompare)) {
-            throw new TypeError(L('There is an identical open proposal.'))
+            throw new Errors.GIErrorUIRuntimeError(L('There is an identical open proposal.'))
```

Comment:
@taoeffect here we have an interesting scenario: when this error happens, it doesn't make sense to ask the user to report us the error because it's a controlled / predictable / custom error.

![image](https://user-images.githubusercontent.com/14869087/78981339-a1f6ed00-7b17-11ea-9313-087994d97574.png)

So, my idea to overcome this problem is to throw a custom GIError instead of the default `TypeError`, so that `LError()` can detect it and not show the link. (See `translations.js`).

I remember a while ago you said we should not use `GIErrorUIRuntimeError` in this file but I don't remember why and now I even thing it's beneficial. Even if there's a problem, we can create another custom error. (a suggestion is welcome).

Anyway, with this solution, now the error is displayed correctly:

![image](https://user-images.githubusercontent.com/14869087/78981859-c69f9480-7b18-11ea-83ff-e9f3eef7f37c.png)

Reply:
@taoeffect now the comment is on the right place!

Reply:
It's really important for these to be TypeErrors in `contracts`... there's code in `state.js` that looks out for that. There's gotta be a better solution here. I don't see why changing this `GIErrorUIRuntimeError` matters, and it needs to be `TypeError` because of `state.js` stuff

Reply:
It can't be `TypeError` so we can filter the type of error on `LError` as I explained on the previous comment. With it, we can choose to not show the "report" link within the error message. 

I'm sorry Greg, what's the problem with `TypeError`? What if we create a new type of error `GiUIErrorContract`, couldn't we modify state to adapt to it? I didn't find any other solution, good or bad.

Reply:
> It can't be TypeError so we can filter the type of error on LError as I explained on the previous comment.

Needs to be a different way of doing it, can't do it this way, and `LError` must not check for error types.

> I'm sorry Greg, what's the problem with TypeError?

See `state.js` and search for `TypeError`

> What if we create a new type of error GiUIErrorContract, couldn't we modify state to adapt to it? 

Nope, doesn't make sense. Contract errors are not UI errors.

Reply:
Feel free to check for the error type in this modal. The contract code uses very specific error types and changing those will break things.

Reply:
> Needs to be a different way of doing it, can't do it this way, and LError must not check for error types.

Any suggestion? I'm not seeing any way of detecting the type of message if we don't throw a custom error.

> Nope, doesn't make sense. Contract errors are not UI errors.

Hum... when we say `There is an identical open proposal.` it sounds very much as an UI error.

Reply:
> Any suggestion? I'm not seeing any way of detecting the type of message if we don't throw a custom error

Yes, feel free to do these checks in the Modal code.

> How do we check the error type if we can't change TypeError?

You can check for TypeError in the Modal, can't you?

> Hum... when we say There is an identical open proposal. it sounds very much as an UI error.

It is. Doesn't mean we need to break the contract code or add unnecessary checks to LError.

Sorry for curt replies right now. I'm hoping you can figure this out while I try to figure out the payments/currency stuff!

Reply:
Yes, I can check the instance of Error on the modal, but for it to work we need to create a custom typeError, like `GIErrorContract`. If we just throw a normal `TypeError`, we can't detect the "custom error".

Reply:
A `TypeError` is a custom error. You can check for it just like `state.js` checks for it

Reply:
Hum... I'll try it out @taoeffect and let you know when I have updates. Thanks and good luck on the payments stuff!

Reply:
@taoeffect, both errors _"Cannot set property 'dumbError' of undefined"._ and _"There is an identical open proposal"_ return true when `instanceof TypeError`, so we cannot distinguish between both in the code, although they are different errors from a UX perspective.

So, since I couldn't find a solution that works in your POV and you don't have time to properly try it in your computer, I suggest to leave it with the simple approach where we always show the link when there's an error and opened an issue (bug) assigned to you to fix it later.

Reply:
@sandrina-p brilliant resolution! 👍 🥇